examples/features/policy-based-routing/config-file-nse.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: policies-config-file
+data:
+  config.yaml: |
+    - from: 172.16.2.201/24
+      proto: 6
```

Comment:
This field is super hard to understand. I think we should consider improving clarity.